ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -272,6 +283,30 @@ open class ShimmerView: UIView {
 		animationGroup.timeOffset = animationSynchronizer?.timeOffset(for: shimmeringLayer) ?? 0
 		shimmeringLayer.add(animationGroup, forKey: "shimmering")
 	}
+
+	private func searchLeaves(in view: UIView, output: inout [UIView]) {
+		for v in view.subviews {
+			if v.subviews.isEmpty && v != self {
+				output.append(v)
+			} else if !v.isHidden {
+				searchLeaves(in: v, output: &output)
+			}
+		}
+	}
+	
+	private static let defaultAlpha: CGFloat = 0.4
+	private static let defaultWidth: CGFloat = 180
+	private static let defaultAngle: CGFloat = -(CGFloat.pi / 45.0)
+	private static let defaultSpeed: CGFloat = 350
+	private static let defaultDelay: TimeInterval = 0.4
+
+	private static let defaultViewTintColor: UIColor = Colors.Shimmer.tint
+	private static let defaultCornerRadius: CGFloat = 4.0
+	private static let defaultLabelCornerRadius: CGFloat = 2.0
+	private static let defaultUsesTextHeightForLabels: Bool = false
+	private static let defaultLabelHeight: CGFloat = 11
+
+	private let defaultShimmersLeafViews: Bool = false
```

Comment:
SwiftLint doesn't like fileprivate, even though that's what our Microsoft Swift Guide says to use. Which do we change?
https://github.com/microsoft/swift-guide/blob/gh-pages/FileContentOrdering.md

Also, I might need to make these public so I can expose them as constants to React Natice

Reply:
yeah update the lint!